modules/nw-ovn-kubernetes-migration-about.adoc

```diff
@@ -18,6 +18,11 @@ A migration to the OVN-Kubernetes network plugin is supported on the following p
 * {rh-virtualization-first}
 * VMware vSphere
 
+[IMPORTANT]
+====
+Migrating to the OVN-Kubernetes network plugin is not supported for the managed service deployments such as OpenShift Dedicated and Red Hat OpenShift on AWS (ROSA).
```

Comment:
Migrating to or from the OVN-Kubernetes network plugin is not supported for the managed OpenShift cloud services such as OpenShift Dedicated and Red Hat OpenShift Service on AWS (ROSA)